python/cudf_polars/tests/testing/test_asserts.py

```
@@ -40,28 +40,32 @@ def test_translation_assert_raises(engine: pl.GPUEngine):
 
     with pytest.raises(AssertionError):
         # This should fail, because we can't translate this query, but it doesn't raise E.
-        assert_ir_translation_raises(unsupported, E)
+        assert_ir_translation_raises(unsupported, engine, E)
 
 
-def test_sink_ir_translation_raises_bad_extension():
+def test_sink_ir_translation_raises_bad_extension(engine: pl.GPUEngine):
     df = pl.LazyFrame({"a": [1, 2, 3]})
     # Should raise because ".foo" is not a recognized file extension
     with pytest.raises(ValueError, match="Unsupported file format: .foo"):
```

Ruff (0.15.13)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)